extensions/amp-social-share/1.0/social-share-config.js

```diff
@@ -36,99 +40,104 @@ let SocialShareConfigDef;
  * contains the following properties:
  *   {string} shareEndpoint - The base API endpoint for sharing to the
  *     specified social media type.
- *   {JsonObject} defaultParams - Parameters to be appended to the end of the
+ *   {Object} defaultParams - Parameters to be appended to the end of the
```

Comment:
Let's move this per-property description to the `SocialShareConfigDef` type.

Reply:
Thanks, updated!